README.md

```diff
@@ -74,7 +74,7 @@ already been discovered:
 | Nimrod        | [&bull;][nim-soln]     |                       |
 | OCaml         | [&bull;][ocaml-soln]   |                       |
 | Objective-J   | [&bull;][obj-j-soln]   |                       |
-| PHP           |                        | [&bull;][php-soln]    |
+| PHP           | [&bull;][php-solv]     | [&bull;][php-soln]    |
```

Comment:
Also, since this solution is specific to PHP7, can you add a new line entry for PHP 7 instead? Just like what was done for Perl 6. Make sure to change the tag name too from `php-soln` to `php7-soln`.